qemu/tests/cfg/block_with_iommu.cfg

```diff
@@ -34,6 +34,8 @@
             type = unattended_install
             only Windows
             only virtio_scsi, virtio_blk
+            # Limits test on Intel CPU only as the bz1974682
```

Comment:
We don't suggest adding the bz info here, instead, we could give a brief summary.
Since the bz info just helps us track this issue, not useful for the upstream users.

Reply:
OK, thank you, I updated it, could you help to check it again? thanks~